components/ActionSheet/docs/typical-use.md

```diff
@@ -0,0 +1,40 @@
+### Typical use: themed buttons
+
+Create an instance of `MDCActionSheetController` and add actions to it, once actions are added, just present it.
```

Comment:
> Create an instance of `MDCActionSheetController` and add actions to it, once actions are added, just present it.

Consider breaking this into multiple sentences and softening the "just present it" with a more neutral "You can now present the action sheet controller."

Reply:
Done